requirements/customer/goals.tex

```diff
@@ -0,0 +1,27 @@
+\section{Purpose}
+  \subsection{Product Goal}
+    JobAdder aims to facilitate and accelerate the execution of computationally expensive jobs in a cluster of work
+    machines.
+
+    Currently, users need to access every single working machine manually in order to distribute their jobs over them.
+    Even more problems arise when a group of users is involved, in that case, users have to look for the working machine
+    with the least amount of work and add their job to it.
+    
+    JobAdder addresses and solves these problems. With JobAdder, users are able to send their jobs to a central 
+    server in order to evenly distribute them over the working machines. So if there's enough workload (which is usually
+    the case), no machine remains idle unless it's asked to.
```

Comment:
I don't think asked is the correct verb for a machine. Maybe forced or sm like that.